### PR TITLE
Create a protocol risks page

### PR DESCRIPTION
**Pull request by pkattera**
# Description

A page to outline risks for LPs on the Balancer Protocol.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality): Enabling anchor links
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [x] Documentation or wording changes
- [ ] Other

## How should this be tested?

Go to the new page at /#/risks (also found in the footer) and make sure it displays well across different screen sizes.

View the two links:
- In the footer
- On the Connect Wallet screen.

One important change to test is the scroll behavior across the app. I made a change to the `scrollBehavior` function in `src/plugins/router/index.ts`, in order to enable normal functioning on anchor links which currently don't seem to work across the site. Please review this code.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/plugins/router/index.ts [hunk 1]
```diff
@@ -205,8 +213,14 @@ if (isGoerli.value) {
 const router = createRouter({
   history: createWebHashHistory(),
   routes,
-  scrollBehavior() {
-    return { top: 0 };
+  scrollBehavior(to, from, savedPosition) {
```
Comment (pkattera) [inline src/plugins/router/index.ts:216]: Please review these changes in detail. I did this to ensure that anchor links would work (previously, anchor links would place users at the top of the page). 

One nice advantage is that people return to the position they were previously at. So if a user on the home page loads some extra rows and scrolls down before clicking in to a pool, they will be returned to the same position if they return back to the homepage. 

Could this have any other unintended consequences?

Comment (alter-eggo) [reply src/plugins/router/index.ts:216]: hm, I think it's ok, but need some testing

File: src/plugins/router/index.ts [hunk 2]
```diff
@@ -205,8 +213,14 @@ if (isGoerli.value) {
...
+    if (savedPosition) {
+      return savedPosition;
+    }
+    if (to.hash) {
+      return { el: to.hash };
+    }
+    return { x: 0, top: 0 };
```
Comment (garethfuller) [inline src/plugins/router/index.ts:223]:
```suggestion
    if (savedPosition) return savedPosition;
    if (to.hash) return { el: to.hash };

    return { x: 0, top: 0 };
```

**Review by garethfuller — CHANGES_REQUESTED** [review]
Not sure I like the smooth scroll behaviour here: https://www.loom.com/share/3f8c7df8bb834781a642358a4baa5c57 On clicking back to the home page should it not just be immediately at the previous location rather than smooth scrolling down to it?

**Review by garethfuller — APPROVED** [review]

**Pull request merged**